docs/_spec/04-basic-definitions.md

```diff
@@ -562,15 +555,15 @@ ParamType          ::=  Type
                      |  Type ‘*’
 ```
 
-A _method declaration_ has the form `def ´f\,\mathit{psig}´: ´T´`, where ´f´ is the method's name, ´\mathit{psig}´ is its parameter signature and ´T´ is its result type.
-A _method definition_ `def ´f\,\mathit{psig}´: ´T´ = ´e´` also includes a _method body_ ´e´, i.e. an expression which defines the method's result.
+An _abstract method definition_ has the form `def ´f\,\mathit{psig}´: ´T´`, where ´f´ is the method's name, ´\mathit{psig}´ is its parameter signature and ´T´ is its result type.
+A _concrete method definition_ `def ´f\,\mathit{psig}´: ´T´ = ´e´` also includes a _method body_ ´e´, i.e. an expression which defines the method's result.
 A parameter signature consists of an optional type parameter clause `[´\mathit{tps}\,´]`, followed by zero or more value parameter clauses `(´\mathit{ps}_1´)...(´\mathit{ps}_n´)`.
-Such a declaration or definition introduces a value with a (possibly polymorphic) method type whose parameter types and result type are as given.
+Such a definition introduces a method with a type (if there is no type or term parameter clause) or methodic type whose parameter types and result type are as given (otherwise).
```

Comment:
This sentence is hard to parse, are we distinguishing "method with a type" versus "method with a methodic type"? Could the former be called "method with a proper type" to avoid confusion?

Reply:
Is it better now?